packages/corelib/src/pubsub.ts

```diff
@@ -0,0 +1,209 @@
+import { DBPart } from './dataModel/Part'
+import { CollectionName } from './dataModel/Collections'
+import { MongoQuery } from './mongo'
+import { AdLibAction } from './dataModel/AdlibAction'
+import { AdLibPiece } from './dataModel/AdLibPiece'
+import { RundownBaselineAdLibAction } from './dataModel/RundownBaselineAdLibAction'
+import { RundownBaselineAdLibItem } from './dataModel/RundownBaselineAdLibPiece'
+import { DBPartInstance } from './dataModel/PartInstance'
+import { DBRundown } from './dataModel/Rundown'
+import { DBRundownPlaylist } from './dataModel/RundownPlaylist'
+import { DBSegment } from './dataModel/Segment'
+import { DBShowStyleBase } from './dataModel/ShowStyleBase'
+import { DBShowStyleVariant } from './dataModel/ShowStyleVariant'
+import { DBStudio } from './dataModel/Studio'
+import { IngestDataCacheObj } from './dataModel/IngestDataCache'
+import { DBTimelineDatastoreEntry } from '@sofie-automation/shared-lib/dist/core/model/TimelineDatastore'
+import { Blueprint } from './dataModel/Blueprint'
+import { BucketAdLibAction } from './dataModel/BucketAdLibAction'
+import { BucketAdLib } from './dataModel/BucketAdLibPiece'
+import { ExpectedMediaItem } from './dataModel/ExpectedMediaItem'
+import { ExpectedPackageWorkStatus } from './dataModel/ExpectedPackageWorkStatuses'
+import { ExpectedPackageDB, ExpectedPackageDBBase } from './dataModel/ExpectedPackages'
+import { ExternalMessageQueueObj } from './dataModel/ExternalMessageQueue'
+import { PackageContainerStatusDB } from './dataModel/PackageContainerStatus'
+import { PeripheralDevice } from './dataModel/PeripheralDevice'
+import { Piece } from './dataModel/Piece'
+import { PieceInstance } from './dataModel/PieceInstance'
+import { TimelineComplete } from './dataModel/Timeline'
+import {
+	RundownId,
+	RundownPlaylistId,
+	ShowStyleBaseId,
+	StudioId,
+} from '@sofie-automation/shared-lib/dist/core/model/Ids'
+import { RundownPlaylistActivationId } from './dataModel/Ids'
+
+export enum CorelibPubSub {
```

Comment:
We should write a desciption for this. Something like "List of the publications used by the Sofie Core GUI" or similar?